plugins/woocommerce/includes/admin/settings/class-wc-settings-general.php

```diff
@@ -188,6 +244,10 @@ protected function get_settings_for_default_section() {
 					),
 				),
 
+				$enable_autocomplete_setting,
```

Comment:
As someone without context, looking at `enable_autocomplete_setting` the name is vague. How do you feel about being explicit and using `address_autocomplete` instead of `autocomplete` in both the variables.

Reply:
I updated all variable names here to be more explicit